README.md

```diff
@@ -4,6 +4,8 @@
 [![License: MIT](https://img.shields.io/github/license/microsoft/semantic-kernel)](https://github.com/microsoft/semantic-kernel/blob/main/LICENSE)
 [![dotnet](https://github.com/microsoft/semantic-kernel/actions/workflows/dotnet-ci.yml/badge.svg?branch=main)](https://github.com/microsoft/semantic-kernel/actions/workflows/dotnet-ci.yml)
 
+> ⚠️ **NOTE**: This project is in early alpha and, just like AI, will evolve quickly. We invite you to join us in developing the Semantic Kernel together!
```

Comment:
Would it be worth adding an ask for them to, "Please contribute by using GitHub Discussions, opening GitHub Issues, sending us PRs, and joining our Discord Channel?"